sklearn/neighbors/base.py

```diff
@@ -332,14 +336,21 @@ class from an array representing our data set and ask who's
                 raise ValueError(
                     "%s does not work with sparse matrices. Densify the data, "
                     "or set algorithm='brute'" % self._fit_method)
-            result = self._tree.query(X, n_neighbors,
-                                      return_distance=return_distance)
+            result = Parallel(n_jobs, backend='threading')(
+                delayed(self._tree.query, False)(
```

Comment:
I would find it more readable if you wrote 'delayed(self._tree.query, check_pickle=False)('

in other words, being explicit on what the 'False' argument means.